tests/test_bounds.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# This script is a draft. Note: Bash is not available on CheriBSD.
+# This is a simple way to test the examples by checking ``stderr``.
+
+: "${SSHPORT:=10021}"
```

Comment:
What does the `:` do here?

Reply:
Assign the default port (the one we have in our repo) in case SSHPORT is not defined in the env.

Reply:
I did not know that!

Reply:
`${parameter:=word}` means set parameter to word if and only if parameter is not already set to a non-empty value. This is great if you have an opinionated $SSHPORT variable already set.